crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -42,6 +44,13 @@ use axum::{
             ACCESS_CONTROL_ALLOW_ORIGIN,
         },
         HeaderValue,
+        Request as AxumRequest,
+        Response as AxumResponse,
```

Comment:
Nit: This is a re-export so I'd alias it to `HttpRequest`, or even better just import `axum::http` and keep the namespacing: `http::Request`.

Reply:
agreed and done: 90c411f54c